.github/workflows/flux-diff.yaml

```diff
@@ -1,160 +1,170 @@
+          path: default/${{ inputs.path }}/${{ inputs.cluster-repo }}
 
       # https://github.com/alexellis/arkade-get
       - name: install flux
-        if: ${{ inputs.flux-cluster-repo }}
-        uses: alexellis/arkade-get@88abff469636be1bd282cff079d75973cdb69cd1
+        if: ${{ inputs.cluster-repo }}
+        uses: alexellis/arkade-get@f577b52ec3feda04af5c36d637eddc1657184b7b
         with:
           print-summary: false
           flux: latest
 
       - name: run docker and push artifact
-        if: ${{ inputs.flux-cluster-repo }}
+        if: ${{ inputs.cluster-repo }}
         run: |
           docker run -d --rm --name kind-registry -p 5050:5000 registry:2.8.1
           flux push artifact oci://localhost:5050/${{ github.event.repository.name }}:local \
-            --path="pull/${{inputs.devenv-oci-path}}" \
+            --path="pull/${{ inputs.path }}" \
             --source="local" \
             --revision="latest"
 
       # https://github.com/allenporter/flux-local
-      - name: diff resources
-        uses: docker://ghcr.io/allenporter/flux-local:v7.10.0@sha256:af0e9be64b5a3384a8f4140d59bf8be876fcc6168091371b98b5e40e111c13d8
+      - name: run flux-local diff
+        uses: docker://ghcr.io/allenporter/flux-local:v8.0.1@sha256:5c8cb0ff9d26a5260a47e7b3949403d80713d80037b9f0e4c02c5efca3588518
         with:
           args: >-
-            diff ${{ matrix.resources }}
-            --unified 6
-            --path /github/workspace/pull/${{ matrix.paths }}/flux
-            --path-orig /github/workspace/default/${{ matrix.paths }}/flux
-            --strip-attrs "helm.sh/chart,checksum/config,app.kubernetes.io/version,chart"
-            --limit-bytes 10000
+            diff ${{ matrix.resource }}
             --all-namespaces
-            --sources ${{ inputs.sources }}
+            --path /github/workspace/pull/${{ inputs.path }}/flux
+            --path-orig /github/workspace/default/${{ inputs.path }}/flux
+            ${{ inputs.sources && format('--sources {0}', inputs.sources) || '' }}
+            --strip-attrs "helm.sh/chart,checksum/config,app.kubernetes.io/version,chart"
+            --unified 6
+            --limit-bytes 9940
             --output-file diff.patch
 
       - name: generate diff
         id: diff
         run: |
-          cat diff.patch;
-          {
-            echo 'diff<<EOF'
-            cat diff.patch
-            echo EOF
-          } >> "$GITHUB_OUTPUT";
-          {
-            echo "### Diff"
-            echo '```diff'
-            cat diff.patch
-            echo '```'
-          } >> "$GITHUB_STEP_SUMMARY"
+          if [ -f diff.patch ] && [ -s diff.patch ]; then
+            echo 'diff<<EOF' >> "$GITHUB_OUTPUT"
+            cat diff.patch >> "$GITHUB_OUTPUT"
+            echo 'EOF' >> "$GITHUB_OUTPUT"
+            {
+              echo "### flux diff (${{ matrix.resource }})"
+              echo "generated at: $(date +'%Y-%m-%d %H:%M:%S %Z')"
```

Comment:
I would use CET, I think this is more readable for us:
`TZ='Europe/Vienna'; echo "generated at: $(date +'%Y-%m-%d %H:%M:%S %Z')"`